src/Core/Types/PropertyType.cs

```diff
@@ -221,8 +222,8 @@ protected override string GetNameInProperFormat(string name, int id)
         {
             StringBuilder sb = new StringBuilder(name);
 
-            sb.Replace("UIA_", "");
-            sb.Replace("PropertyId", "");
+            sb.Replace(DisplayStrings.UIA_, "");
+            sb.Replace(DisplayStrings.PropertyId, "");
```

Comment:
These are used for mapping to a code construct. I would not recommend moving them into a resource file